src/aequitas/bias.py

```diff
@@ -11,52 +16,96 @@
 class Bias(object):
     """
     """
-    def __init__(self, key_columns=None, input_group_metrics=None, fill_divbyzero=None):
+    default_key_columns = ('model_id', 'score_threshold', 'attribute_name')
+    all_group_metrics = ('ppr', 'pprev', 'precision', 'fdr', 'for', 'fpr',
+                         'fnr', 'tpr', 'tnr', 'npv')
+    non_attr_cols = ('score', 'model_id', 'as_of_date', 'entity_id', 'rank_abs',
+                     'rank_pct', 'id', 'label_value')
+    significance_measures = ('fpr', 'fnr')
+
+    def __init__(self, key_columns=default_key_columns, sample_df=None,
+                 non_attr_cols=non_attr_cols,
+                 significance_cols=significance_measures,
+                 input_group_metrics=all_group_metrics, fill_divbyzero=None):
         """
 
         :param key_columns:
         :param input_group_metrics:
         :param fill_divbyzero:
         """
-        if not key_columns:
-            self.key_columns = ['model_id', 'score_threshold', 'attribute_name']
-        else:
-            self.key_columns = key_columns
-        if not input_group_metrics:
-            self.input_group_metrics = ['ppr', 'pprev', 'precision', 'fdr', 'for', 'fpr', 'fnr', 'tpr', 'tnr', 'npv']
-        else:
-            self.input_group_metrics = input_group_metrics
+        self.key_columns = list(key_columns)
+        self.input_group_metrics = list(input_group_metrics)
+
         if not fill_divbyzero:
             self.fill_divbyzero = 10.00000
         else:
             self.fill_divbyzero = fill_divbyzero
+        self.non_attr_cols = non_attr_cols
+        self.significance_cols = significance_cols
+        # if sample_df:
+        #     self.samples = self.get_measure_sample(sample_df, attribute, measure)
```

Comment:
Is this commented-out block something you can now remove? (Usually there's little utility in committing this sort of thing.)